.github/workflows/pull_request_ci.yml

```
@@ -0,0 +1,35 @@
+name: Pull request checks
+
+on:
+  pull_request:
+
+jobs:
+  check-unique-standard-names:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.x"
+
+      - name: Check for duplicate standard names
+        run: python3 check_xml_unique.py standard_names.xml
+
+  check-rerendered-markdown:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.x"
+
```

We'll have to add the `xmllint` install step here as well:

```suggestion
 # Install xmllint so tests pass:
      - name: Install xmllint
        run: |
            sudo apt-get update
            sudo apt-get install libxml2-utils
```